### PR TITLE
[4.17] Add conformance periodic for hypershift kubevirt on azure infra

### PR DESCRIPTION
**Pull request by orenc1**

### CODE REVIEW

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by davidvossel** [comment]
@orenc1 maybe we want to drop this now that the azure presubmit isn't performing as well as we hoped?

**Comment by orenc1** [comment]
> @orenc1 maybe we want to drop this now that the azure presubmit isn't performing as well as we hoped?

I believe the conformance suite can be passed on azure infra.
the discrepancies are not that big.
I'm rebasing an rerunning to get an up-to-date picture of the situation.

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-azure

**Comment by orenc1** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-azure

File: ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.17-periodics.yaml [hunk 1]
```diff
@@ -1016,6 +1016,87 @@ periodics:
...
+    org: openshift
+    repo: hypershift
+  labels:
+    ci-operator.openshift.io/cloud: azure4
+    ci-operator.openshift.io/cloud-cluster-profile: azure4
+    ci-operator.openshift.io/variant: periodics
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-kubevirt-azure
```
[the remark this answers is not in the document]
Comment (orenc1) [reply ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.17-periodics.yaml:1036]: done.

**Pull request merged**